nexus/base/api/routers.py

```diff
@@ -29,9 +29,10 @@ class SingletonRouter(routers.SimpleRouter):
         # Dynamically generated list routes.
         # Generated using @list_route decorator
         # on methods of the viewset.
-        routers.DynamicListRoute(
-            url=r'^{prefix}/{methodname}{trailing_slash}$',
-            name='{basename}-{methodnamehyphen}',
+        routers.DynamicRoute(
+            url=r'^{prefix}/{url_path}$',
```

Comment:
In this case,  how do we accomdate `methodname`. Can you pass on the links to docs?

Reply:
We don't need to use `methodname` explicitly, `url_path` would suffice. [source](https://www.django-rest-framework.org/api-guide/routers/#customizing-dynamic-routes)